### PR TITLE
enh: add scope for show xx variables

### PR DESCRIPTION
**Pull request by glzhao89**

### CODE REVIEW

File: source/common/src/tglobal.c [hunk 1]
```diff
@@ -340,214 +340,213 @@ static int32_t taosAddClientCfg(SConfig *pCfg) {
...
-  if (cfgAddInt64(pCfg, "queryMaxConcurrentTables", tsQueryMaxConcurrentTables, INT64_MIN, INT64_MAX, 1) != 0)
-    return -1;
-  if (cfgAddInt32(pCfg, "metaCacheMaxSize", tsMetaCacheMaxSize, -1, INT32_MAX, 1) != 0) return -1;
-  if (cfgAddInt32(pCfg, "slowLogThreshold", tsSlowLogThreshold, 0, INT32_MAX, true) != 0) return -1;
-  if (cfgAddString(pCfg, "slowLogScope", "", true) != 0) return -1;
+  if (cfgAddString(pCfg, "firstEp", "", CFG_SCOPE_BOTH) != 0) return -1;
+  if (cfgAddString(pCfg, "secondEp", "", CFG_SCOPE_BOTH) != 0) return -1;
+  if (cfgAddString(pCfg, "fqdn", defaultFqdn, CFG_SCOPE_SERVER) != 0) return -1;
+  if (cfgAddInt32(pCfg, "serverPort", defaultServerPort, 1, 65056, CFG_SCOPE_SERVER) != 0) return -1;
+  if (cfgAddDir(pCfg, "tempDir", tsTempDir, CFG_SCOPE_SERVER) != 0) return -1;
```
Comment (dapan1121) [inline source/common/src/tglobal.c]: ?

File: source/common/src/tglobal.c [hunk 2]
```diff
@@ -340,214 +340,213 @@ static int32_t taosAddClientCfg(SConfig *pCfg) {
...
-
-  if (cfgAddInt64(pCfg, "openMax", tsOpenMax, 0, INT64_MAX, 1) != 0) return -1;
+  if (cfgAddTimezone(pCfg, "timezone", tsTimezoneStr, CFG_SCOPE_BOTH) != 0) return -1;
+  if (cfgAddLocale(pCfg, "locale", tsLocale, CFG_SCOPE_BOTH) != 0) return -1;
+  if (cfgAddCharset(pCfg, "charset", tsCharset, CFG_SCOPE_BOTH) != 0) return -1;
+  if (cfgAddBool(pCfg, "assert", 1, CFG_SCOPE_BOTH) != 0) return -1;
+  if (cfgAddBool(pCfg, "enableCoreFile", 1, CFG_SCOPE_BOTH) != 0) return -1;
+  if (cfgAddFloat(pCfg, "numOfCores", tsNumOfCores, 1, 100000, CFG_SCOPE_BOTH) != 0) return -1;
+
+  if (cfgAddBool(pCfg, "SSE42", tsSSE42Enable, CFG_SCOPE_BOTH) != 0) return -1;
```
Comment (dapan1121) [inline source/common/src/tglobal.c:403]: ?

File: source/common/src/tglobal.c [hunk 3]
```diff
@@ -340,214 +340,213 @@ static int32_t taosAddClientCfg(SConfig *pCfg) {
...
+  if (cfgAddInt32(pCfg, "monitorPort", tsMonitorPort, 1, 65056, CFG_SCOPE_SERVER) != 0) return -1;
+  if (cfgAddInt32(pCfg, "monitorMaxLogs", tsMonitorMaxLogs, 1, 1000000, CFG_SCOPE_SERVER) != 0) return -1;
+  if (cfgAddBool(pCfg, "monitorComp", tsMonitorComp, CFG_SCOPE_SERVER) != 0) return -1;
 
-  if (cfgAddBool(pCfg, "crashReporting", tsEnableCrashReport, 0) != 0) return -1;
-  if (cfgAddBool(pCfg, "telemetryReporting", tsEnableTelem, 0) != 0) return -1;
-  if (cfgAddInt32(pCfg, "telemetryInterval", tsTelemInterval, 1, 200000, 0) != 0) return -1;
-  if (cfgAddString(pCfg, "telemetryServer", tsTelemServer, 0) != 0) return -1;
-  if (cfgAddInt32(pCfg, "telemetryPort", tsTelemPort, 1, 65056, 0) != 0) return -1;
+  if (cfgAddBool(pCfg, "crashReporting", tsEnableCrashReport, CFG_SCOPE_SERVER) != 0) return -1;
```
Comment (dapan1121) [inline source/common/src/tglobal.c]: ?

File: source/common/src/tglobal.c (hunk 1 above)
Comment (glzhao89) [reply source/common/src/tglobal.c]: fixed

File: source/common/src/tglobal.c (hunk 2 above)
Comment (glzhao89) [reply source/common/src/tglobal.c:403]: fixed

File: source/common/src/tglobal.c (hunk 3 above)
Comment (glzhao89) [reply source/common/src/tglobal.c]: fixed

**Review by dapan1121 — APPROVED** [review]

**Pull request merged**